.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+# Contributing
+ 
```

Comment:
I would include some intro that we are OSS project, maybe with used licenses explanation.

Then a couple of words about in lines like: "'We like contributions, please help us make a project better..."